presence-firestore/public/index.js

```diff
@@ -21,7 +21,8 @@ function rtdb_presence() {
 
     // Create a reference to this user's specific status node.
     // This is where we will store data about being online/offline.
-    var userStatusDatabaseRef = firebase.database().ref('/status/' + uid);
+    // `push()` will add a new key to the user's path, acting as a deviceId
+    var deviceStatusDatabaseRef = firebase.database().ref('/status/' + uid + '/devices').push();
```

Comment:
`sessions` or `connections` is probably more accurate than `devices`, since the id isn't based on the device id.